packages/mdc-tab-scroller/foundation.ts

```diff
@@ -238,27 +254,25 @@ export class MDCTabScrollerFoundation extends MDCFoundation<MDCTabScrollerAdapte
   }
 
   /**
-   * Internal increment scroll method
-   * @param scrollX The new scroll position increment
+   * Internal method to compute the increment scroll operation values.
+   * @param scrollX The desired scroll position increment
+   * @return MDCTabScrollerAnimation with the sanitized values for performing
+   *     the scroll operation.
    */
-  private incrementScroll_(scrollX: number) {
+  private getIncrementScrollOperation_(scrollX: number):
+      MDCTabScrollerAnimation {
```

Comment:
This can be on the same line as the signature. Was there an autoformatter that pushed it to the next line?

Reply:
Correct, the internal one did so. Should I still put it back?

Reply:
I just went ahead and put it back since it didn't make sense given the rest of the file's line lengths.